mappers/modbus/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM ubuntu:16.04
+
+RUN mkdir -p kubeedge
+
+COPY . kubeedge/
```

Comment:
Maybe need  compilation step in dockerfile.

Reply:
well,I think we should restructure the dockerfile,we can use apline to replace ubuntu to make a more smaller image what's more just add a bin into image is better than add all files into images. WDYT

Reply:
Thanks for the comments.
Will refine it.